### DISCUSSION TITLE
Boot loop due to app updatenotification after migration

### DISCUSSION THREAD

**Discussion by notEvil** [nextcloud/all-in-one#2928] [Questions] [answered]
### Steps to reproduce
1. Migrate from another instance with app updatenotification installed
2. Start container (last step of migration)

### Expected behavior

Nextcloud container starts

### Actual behavior

Nextcloud container tries to upgrade updatenotification but fails because the files were removed.

### Host OS

Arch Linux

#### Nextcloud AIO version

6.2.1

#### Current channel

latest

#### Other valuable info

Same as #1164 which was supposed to be fixed. After I removed the corresponding rows in oc_appconfig (in SQL dump), the container did boot successfully. There was very little output unrelated to this issue in the logs (didn't keep them unfortunately), so I guess the upgrade started before the fix could be applied.

**Comment by notEvil** [comment, marked as answer]
I just failed to reproduce the issue on a different machine using the same db dump, `data/appdata_...` and `data/updater-...`. So I probably messed up the migration somehow (e.g. app version differences) which led to the upgrade behavior.

**Comment by szaimen** [comment]
All right! Thanks for verifying!
